src/test/kotlin/org/openmbee/mms5/util/Requests.kt

```diff
@@ -66,6 +69,30 @@ fun TestApplicationRequest.setSparqlQueryBody(body: String) {
 }
 
 fun TestApplicationEngine.httpRequest(method: HttpMethod, uri: String, setup: TestApplicationRequest.() -> Unit): TestApplicationCall {
+    if("1" != System.getProperty("MMS5_TEST_NO_AUTH", "")) {
+        handleRequest(method, uri) {
+            addHeader("Authorization", authorization(anonAuth))
+            setup()
+        }.apply {
+            if("" != System.getProperty("MMS5_TEST_EXPECT", "")) {
+                response shouldHaveStatus System.getProperty("MMS5_TEST_EXPECT").toInt()
+            }
+            else {
+                if(200 === response.status()?.value) {
+                    println("?");
+                }
+
+                response shouldHaveOneOfStatuses setOf(HttpStatusCode.NotFound, HttpStatusCode.Forbidden)
```

Comment:
when would it be not found? if the user/group doesn't exist?

Reply:
404 is a valid response code in practice a user does not have auth. it's like "neither confirm nor deny" the existence of something